Make password view when email is present

renancarvalho opened the pull request:
<!--
Link a related issue to this PR. If the PR does not immediately resolve the issue,
for example, it requires a separate deployment to production, avoid
using the "fixes" keyword and instead attach the [Status] Fix Inbound label to
the linked issue.
-->

Related to #53495

## Proposed Changes

When loading the login form, checks if there's email or username filled, if so, load the form with password field visible.
This is helpful in this https://github.com/Automattic/wp-calypso/issues/53495 

## Testing Instructions

<!--
Add as many details as possible to help others reproduce the issue and test the fix.
"Before / After" screenshots can also be very helpful when the change is visual.
-->

* Pull and run this branch or use calypso live link
* Navigate using incognito or logged-out user to /log-in
* Fill your email or username and click in continue
* In the next step, click in "Email a login link"
* Click on “enter a password instead”
* You should see the login for with the password field enabled.

## Pre-merge Checklist

<!--
Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.
-->

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

renancarvalho committed:
Make password view when email is present

renancarvalho committed:
Make password view when email is present

renancarvalho committed:
Make password view when email is present

artpi committed:
Big Sky: Set expectations in the prompt copy (#83763)

* wording changes

* Neel's copy

* remove label

manzoorwanijk committed:
Social previews | Add Nextdoor preview (#83746)

* Add Nextdoor icons

* Add Nextdoor post preview

* Add link preview

* Expose preview components

* Update CHANGELOG.md

* Wire up Nextdoor preview for Calypso posts UI

* Bump version

renancarvalho committed:
Validade account type

renancarvalho committed:
Validade account type

renancarvalho committed:
Validade account type

southp committed:
Fix the condition for setting the intent state value and for showing (#83798)

the loading spinner

aneeshd16 committed:
Plans Comparison: add sticky behaviour to the header bar (#83042)

* Plans Comparison: add sticky behaviour to the header bar

* [plan-features] Remove unnecessary translationY transform in plans-wrapper and is-section-plans

* Remove unnecessary min-height CSS property

* Increase z-index on sticky-container

* Check for IntersectionObserver reliably

* Fix StickyContainer logic to handle browsers without IntersectionObserver support.

alshakero reviewed (changes requested):
Hmmm, not sure this is the right fix, it acts strange when the account is passwordless. Since you’re fetching the account type, maybe wait until you know it’s not passwordless before showing the password?

Addison-Stavlo committed:
Reader - Logged out /read - Rediect to /discover instead of signup page. (#83769)

* redirect logged out /read to /discover instead of signup

* remove unnecessary declaration

Addison-Stavlo committed:
add #comments id to post url when redirecting logged out user to full post from a comment context (#83813)

ddc22 committed:
Remove the is skinned flag from grid related code (#83473)

Initsogar committed:
ActivityLog: Decouple filterbar from the rendering logic (#83780)

- Move the filterbar rendering logic to its own renderFilterbar() function
- Validate filterbar loading state using `isRequestingSiteFeatures`

rcrdortiz committed:
Theme Discovery: Restore Theme Showcase row elements to match production (#83803)

* Restored the default theme showcase views to tnot having the same elements as the collection view

* The collection view will continue to display 3 items per row

* Removed className from the Theme Showcase.

rcrdortiz committed:
Theme Discovery: Remove Discovery view and default back to Recommended view (#83799)

* Removed unused import

* The collection view header is now styled properly

dsas committed:
Fix the block-editor Manage Pattterns menu (#83474)

The block editor menu has a Manage Patterns link, following it on dotcom
when using the iframed post editor results in an iframe warning message.

This change makes it work again.

dognose24 committed:
Subscriber Stats: Use null values for points before table creation (#83816)

alshakero committed:
Revert "ETK Performance: defer loading the Help Center" (#83820)

alshakero committed:
Revert "ETK Performance: defer loading the domain upsell callout" (#83819)

Co-authored-by: Louis Laugesen <louis.laugesen@gmail.com>

simison committed:
Subscriber management: link to stats instead of subscriber block docs (#83423)

AtrumGeost committed:
Update/2023 dotcom division meetup support closures for Quick Start support (#83784)

* Modify message and dates for QS sessions

* Add breakline for better visualization

delputnam committed:
Add support for ALIAS records (#83490)

* Add support for ALIAS records

* Update ALIAS record definition text

niranjan-uma-shankar committed:
Add missing / for the URL (#83818)

escapemanuele committed:
Signup: handle passswordless existing accounts (#83767)

DustyReagan committed:
Multi Domains: Disable the experiment and enable multi-domains on onboarding (#83824)

* Remove experiment and enabled on onboarding flow

* Remove currentUser parameter from shouldUseMultipleDomainsInCart

* Add comment

Initsogar committed:
ActivityLog: display number of activities selected when `showAppliedFiltersCount` prop is passed (#83786)

* Display number of types selected when necessary

- Add a new showAppliedFiltersCount prop to `TypeSelector`
- Move render logic for selector button to its own function

* Add translations to activities selected text

* Replace activity type title to sentence case

* Display filters count when more than 1 is selected

sirbrillig committed:
Checkout: reorganize line items and summary for new design behind flag (#83204)

* Render order review in sidebar if query string is set

* Consolidate checkout summary line items

* Add SitePreview to checkout sidebar

* Update billing line item styling

* Hide feature summary section

* Update getLabel to remove branding

* Update styles for ItemVariationPicker

* Update ItemVariationPicker term text

* Update and expand SitePreview props and styles

* Update font styling and spacing for item variant

* Remove NonProductLineItems and decrease padding

* First pass new upgrade CTA

* Add CostOverrides component and styling

* Add bottom margin to coupon area

* Add hasCheckoutVersion function to wpcom-checkout package

* Remove inadvertent tsconfig changes

* Add checkoutVersionChecker to wpcom-checkout package

* More tsconfig changes

* Type CostOverrides component

* Add feature flag for checkout-version

* Set development feature flag to true

* Move docBlock for hasCheckoutVersion

* Undo string replacement in label

* Set new SitePreview props as optional

* Pass responseCart.currency to formatCurrency

* Refactor BillingLine styles and render code

* Small spacing changes

* Fix incorrect spacing

---------

Co-authored-by: Chris Diehl <jjchrisdiehl@gmail.com>

paulopmt1 committed:
Added confetti effect on domain transfer thank you page (#83830)

lsl committed:
Improve header margins on mobile screen sizes (#83839)

lsl committed:
Update/likes header (#83841)

* Fix class copy paste

* Likes page padding

jkguidaven committed:
Jetpack Manage: Fix internal links causing a full page reload. (#83795)

lsl committed:
Update/reader lists header (#83836)

* Update reader list header to use navigation header component

* Replace edit button with real buttons

* Remove x-axis padding for reader lists

* Mobile improvements

arthur791004 committed:
Assembler: Update the CTA copy on the Styles screen (#83838)

candy02058912 committed:
Remove exessive padding for /plans for mobile (#83843)

fushar committed:
Assembler: use the perf test endpoint when flag is true (#83840)

southp committed:
The hiding free subdomain test on the paid media flow – 2nd attempt (#83793)

* Update the ExPlat instance slug and preload it in `componentDidMount()`

* Remove the unnecessary preloading.

lsl committed:
Conversations header fixes - css leaks to search (#83854)

lsl committed:
Padding for navigation header on mobile discover page in the reader (#83855)

matticbot committed:
chore(deps): update wordpress monorepo (#83832)

* chore(deps): update wordpress monorepo

* Bump some packageExtensions versions

---------

Co-authored-by: Renovate Bot <bot@renovateapp.com>
Co-authored-by: Marin Atanasov <tyxla@abv.bg>

tyxla committed:
Revert "chore(deps): update wordpress monorepo (#83832)" (#83856)

This reverts commit 816cd852cdf4bd80a4331f60ff13a66d151282ff.

manzoorwanijk committed:
Fix Nextdoor connection description when connected (#83844)

* Fix Nextdoor connection description when connected

* Fix icon color on post list share section

* Fix icon border radius

* Update description

spsiddarthan committed:
Upgrade calypso upgrade map (#83812)

chriskmnds committed:
FoldableCard to packages (Step 3): Update references to use new package component (#83235)

yuliyan committed:
calypso-e2e: Update translateFromPage utility to accept context (#83860)

enejb committed:
Add/app promo to login link (#83135)

* App Promo

Add new options and QR Code component

* Add App Promo to Magic login

* Update get Apps mobile download page to use the new QR Code

* Fix Mobile design

* Fix styling.

* Update client/blocks/app-promo/index.jsx

Co-authored-by: Omar Alshaker <omar@omaralshaker.com>

* Update client/blocks/app-promo/qr-code.jsx

Co-authored-by: Omar Alshaker <omar@omaralshaker.com>

* Fixes

* Update to use tsx files

* Mark things as optional

* Update the example and readme

* Fix campaign and source in app badges

* Minor styling update

* 30 chr max limit for app store.

---------

Co-authored-by: Omar Alshaker <omar@omaralshaker.com>

chriskmnds committed:
FoldableCard to packages (Step 4): Remove FoldableCard component shim (#83237)

DustyReagan committed:
Remove special rule for domain only flow (#83831)

dsas committed:
Switch the default category for build intent (#83865)

The 'Promote myself or business' and 'Other' goals in onboarding are
collectively the 'build' intent. On the design picker during onboarding
this was defaulted to the 'Portfolio' design category since #80751.
Staff feedback since then has been consistently against this.

This change will use the 'Business' category rather than the 'Portfolio'
category

jkguidaven committed:
Jetpack Manage: Fix the issue with the sidebar closing when clicking the search bar. (#83845)

* Hide SitePicker in MySitesNavigation to avoid event handler conflicts with the new sidebar.

* rename 'showSitePicker' to 'renderSitePicker' for better readability.

paulopmt1 committed:
Fixed button position on the managing plans section (#83864)

* Fixed button position on managing plans section

* Merged class styles and removed uncessary div/class

wojtekn committed:
Staging Sites: Disable site database sync for production site with Woo (#83802)

* Disable site database sync for production site with Woo

* Add test for new condition

* Ensure that new constant is boolean

* Adjust new label wording

rcrdortiz committed:
Theme Discovery: Add Partner collection carousel to Recommended view (#83814)

* Removed unused import

* Changed import order.

* Restored CSS class that correctly positions the Partner Collection Carousel

* Removed left and right margin from mobile breakpoints so that the Collection does not overflow the container

* Updated short title to include themes in it

* Fixed some navigation issues

* Removed the Partner Collection Carousel from the All category

escapemanuele committed:
Login: Direct to login link page if user not allowed (#83704)

AllTerrainDeveloper committed:
[Odie] UX Improvements (#83825)

ddc22 committed:
Shopping cart provider not required to the grid (#83561)

WBerredo committed:
Partner Theme: Allow activation of installed theme (#83777)

* Partner Themes: Show activate option when the theme is installed

* Hide banner when the Partner theme is already installed

* Show the Activate option when the theme is installed

sdixon194 committed:
Check if the emailInputLocator exists before trying to fill it (#83879)

sdixon194 committed:
E2E Test - Update Subscribe Block for New Behavior Take Two (#83883)

* Fix the broken subscription test, take two

* Update packages/calypso-e2e/src/lib/blocks/block-flows/subscribe.ts

Co-authored-by: Brad Jorsch <anomiex@users.noreply.github.com>

---------

Co-authored-by: Brad Jorsch <anomiex@users.noreply.github.com>

elliottprogrammer committed:
Pricing page: Fix Jetpack Search product CTA (#83810)

bcotrim committed:
add ai response to email ticket (#83717)

* add ai response to email ticket

* change ticket field name ai_id to ai_chat_id

delputnam committed:
Set TXT record limit to 2048 characters (#83887)

* Set TXT record limit to 2048 characters

* Use <= 2048 vs < 2049

grzegorz-cp committed:
Stats: Fix a stuck loading indicator (#83834)

DustyReagan committed:
Add key to RegisterDomainStep (#83884)

p-jackson committed:
Disable the plans/hosting-trial flag in development (#83783)

creativecoder committed:
Updates blogging prompts to use v3 endpoint (#76518)

* Updates blogging prompts to use v3 endpoint
* Use relative link for prompt tag page
* Ensure most recent prompts for each day are shown first
* Add bloganuary customization to blogging prompts card (#83696)

---------

Co-authored-by: Eru Penkman <roo2@users.noreply.github.com>

yashwin committed:
Jetpack Manage: Fix page break when the site is expanded & minor UI issue. (#83899)

* Check if the object exists to get the backup text to prevent page breaks

* Fix sticky header button spacing

matticbot committed:
chore(deps): update linters (#83895)

Co-authored-by: Renovate Bot <bot@renovateapp.com>

matticbot committed:
chore(deps): update dependency @types/redux-mock-store to v1.0.5 (#83893)

Co-authored-by: Renovate Bot <bot@renovateapp.com>

mpkelly committed:
Unverified domain email address: show a notice on the relevant screens (#83756)

* Suspended Domains: show notice on domain contact details page

* Fix typo

* Undo fix :)

* Restyle notice to make it more prominent

* Fix type error

* Remove warning from edit screen.

* Make notice a warning

* Updated notice text

candy02058912 committed:
Remove margin for NavigationHeader and fix /plans Header (#83909)

mpkelly committed:
Home screen: show notice if domain email is unverified (#83826)

renancarvalho committed:
Check user type

renancarvalho committed:
Check user type

renancarvalho committed:
Merge branch 'fix_login_password' of github.com:Automattic/wp-calypso into fix_login_password

renancarvalho committed:
Fix regular user case

renancarvalho committed:
Fix regular user case

ilyasfoo committed:
Remove free domain copy from mobile view in plans page for Woo Express  offer (#83859)

renancarvalho committed:
Merge branch 'fix_login_password' of github.com:Automattic/wp-calypso into fix_login_password

renancarvalho commented:
Created a new one https://github.com/Automattic/wp-calypso/pull/83916 Closing this one
